clr-k8s-examples/setup_system.sh

```diff
@@ -10,6 +10,14 @@ ADD_NO_PROXY+=",$(hostname -I | sed 's/[[:space:]]/,/g')"
 sudo -E swupd update
 sudo -E swupd bundle-add cloud-native-basic storage-utils
 
+# Add registries for k8s onto crio's registries.conf
+if [ ! -f /etc/containers/registries.conf ]; then
+  cat << EORF > sudo bash -c "cat > /etc/containers/registries.conf"
```

Comment:
@ganeshmaharaj Whats happening really here

Reply:
should be fixed now!